lib/rpush/daemon/dispatcher/http.rb

```diff
@@ -5,7 +5,7 @@ class Http
         def initialize(app, delivery_class, options = {})
           @app = app
           @delivery_class = delivery_class
-          @http = Net::HTTP::Persistent.new('rpush')
+          @http = Net::HTTP::Persistent::VERSION < '3' ? Net::HTTP::Persistent.new('rpush') : Net::HTTP::Persistent.new(name: 'rpush')
```

Comment:
Probs want to use something like this instead of relying on string comparison: https://medium.com/little-programming-joys/how-to-compare-version-strings-in-ruby-ced0916a4b71

Reply:
good call.